tests/virtual_machines_test.go

```diff
@@ -32,6 +34,26 @@ var _ = Describe("Virtual Machines", func() {
 		Expect(result.Error()).NotTo(HaveOccurred())
 	})
 
+	BeforeEach(func() {
+		// Verify that there are no VMs in the cluster
+		currentVMList := &kubevirtv1.VirtualMachineList{}
```

Comment:
This is redundant. we are already checking the vm's were deleted in AfterEach.
Please consider moving all to BeforeEach or the other way around..

Reply:
Its make sense to me that AfterEach should clean after the test, in this case delete the VMs that were created during the test, 
and BeforeEach checks that "we start fresh" and the vmWaitConfigmap is empty.
Maybe we can move the map clearing to AfterEach and leave all the checks in BeforaEach.

I understand it is redundant, but think about it if it was there earlier we were not in this situation ;)

Reply:
I don't think there is a need for a double check. If I understand correctly, this expect could never be true, and therefor redundant. 

The reason that it was missed before was the fact that we had remnant in the configmap, not the vms.

Reply:
In case AfterEach delete VM part which inside an Eventually times out, a VM might be left in the cluster, then this double check will raise an error about it.

I meant, if there was a check or cleaning remnants in the vmWait configmap we were not in this state.